.github/dependabot.yml

```diff
@@ -35,6 +35,10 @@ updates:
       xterm:
         patterns:
           - "@xterm/*"
+    ignore:
+      # https://github.com/cockpit-project/cockpit/issues/21151
+      - dependency-name: "sass"
+      - versions: ["1.80.x", "2.x"]
```

Comment:
The commit message is wrong (and I think that's my fault for confusing you).

The breaking change would be introduced in `2.x` and 1.80.x` started introducing the warnings. So we should allow 2.x and change the commit message.